popup.js

```diff
@@ -0,0 +1,152 @@
+        <li tabi="56" id="c-tag"><a href="#" class="tag" id="css"></a>${workse[i].tags[1]}</li>
+        <li tabi="57" id="j-tag"><a href="#" class="tag" id="JavaScript"></a>${workse[i].tags[2]}</li>
+      </ul>
+    </div>
+    <div class="actions">
+      <button class="NormalB">
+      </button>
+    </div>
+    </div>`;
+}
+const popup = document.createElement('div');
+detailsc.append(popup);
+popup.className = 'details';
+popup.innerHTML = `
+<header class="header-4">
+<div class="top-p">
+  <h2 class="project-tittle"></h2>
+  <button class="btn-close2" alt="close menu"><img src="img/cros.png" class="img-close2"></button>
+</div>
+<ul class="frame">
+  <li class="txt" id="canopy"></li>
+  <li class="img-container"><img src="img/Counter.png" alt="point" id="img-point"></li>
+  <li class="txt" id="role"></li>
+  <li class="img-container"><img src="img/Counter.png" alt="point" id="img-point"></li>
+  <li class="txt" id="year"></li>
+</ul>
+</header>
+<div class="screen-shotp"></div>
+<div class="body-2">
+<div>
+<p class="descriptionm">
+Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+    </p>
+<p class="descriptiondsk">
+  Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+  <br>
+  <br>
+  <br>
+  Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.
+    </p>
+</div>
+<div class="buttons">
+  <ul id="tags-list">
+    <li id="h-tag"><a href="#" class="tag" id="html">HTML</a></li>
+    <li id="c-tag"><a href="#" class="tag" id="css">CSS</a></li>
+    <li id="j-tag"><a href="#" class="tag" id="JavaScript">JavaScript</a></li>
+  </ul>
+  <ul id="tags-list2">
+    <li id="r-tag"><a href="#" class="tag" id="ruby">Ruby on rails</a></li>
+    <li id="b-tag"><a href="#" class="tag" id="ruby">Bootstrap</a></li>
+  </ul>
+  <div class="sbtn">
+  <a href="#">    
+    <button class="LiveB">
+    <img src="img/live.png" alt="live-icon" class="live-img">
+    <p id="sbutton"></p>
+    </button>
+  <a/>
+  <a href="#"> 
+    <button class="gitB">
+    <img src="img/git.png" alt="github-icon" class="live-img">
+    </button> 
+  <a/>
+  </div>
+</div>
+</div>`;
```

Comment:
- [ ] kindly consider adding the pop-up on each project of your portfolio not hard coding a single pop-up for only the first `see project` button and doing so will help you to not miss out on any point in the below 👇  requirement steps 
- You need to use a JavaScript array to store all of the information for all projects.(which you did 👍 )
 - For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.(which you did 👍 )
- You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.(which you did 👍 )
-  After that, you could implement the popup window.(which you are missing out for each see project ❌  )
**suggestion:** You may do so in your CSS by not displaying all the data of each object on the project itself and add a class to display all data on the pop-up instead
doing that will make your code dry cause you won't be repeating yourself

Reply:
>  - [ ] kindly consider adding the pop-up on each project of your portfolio not hard coding a single pop-up for only the first `see project` button and doing so will help you to not miss out on any point in the below 👇  requirement steps 
> - You need to use a JavaScript array to store all of the information for all projects.(which you did 👍 )
>  - For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.(which you did 👍 )

> - You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.(which you did 👍 )
> -  After that, you could implement the popup window.(which you are missing out for each see project ❌  )
> **suggestion:** You may do so in your CSS by not displaying all the data of each object on the project itself and add a class to display all data on the pop-up instead
> doing that will make your code dry cause you won't be repeating yourself
> 
> _Originally posted by @divinecharlotte in https://github.com/CarlosZ96/Carlos-Portfolio/pull/13#discussion_r1173951111_
>     


- As requested by the previous reviouwer, 🙃  When the user clicks the button to check project details, a particular popup with details about that project should appear and not a common one. Kindly address this !!